volttrontesting/platform/test_certs.py

```diff
@@ -1,10 +1,12 @@
 import os
 import tempfile
 import shutil
-
 import pytest
+import json
+from pathlib import Path
 from volttron.platform.certs import Certs, Subject, CertError
 from volttron.platform.agent.utils import get_platform_instance_name
+from volttrontesting.platform.test_certs_utils import *
```

Comment:
perfer not to import * anywhere but in __init__.py files.  better to use explicit import